content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Reset'
+Description: 'Used to undo commits to a Git repository.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the `reset` command is used to change the state of the Git repository or undo commits.
+
+## Syntax
+```
+git reset <mode-option> <commit>
+```
+The mode refers to the options for the reset command. These include:
+- --soft
+- --mixed
+- --hard.
+
+The commit can either be the commit hash generated by Git, or in reference to the HEAD.
+
+## Example
+
+Suppose a file named **example.md** is accidentally committed to a Git repository. To undo the commit, the command `git reset --mixed HEAD~1` is used in the terminal. `--mixed` is the default mode, so `git reset HEAD~1` will achieve the same result. This resets the HEAD and index to the previous commit.
+
+**Note:** HEAD refers to the current commit while index refers to the staging area where files are [added using Git](https://www.codecademy.com/resources/docs/git/add).
+
+## After Committing
+
+```
+On branch main
+Your branch is up to date with 'origin/main'.
+nothing to commit, working tree clean
+```
+
+## After Resetting
+
+```
+On branch main
+Your branch is behind 'origin/main' by 1 commit, and can be fast-forwarded.
+  (use "git pull" to update your local branch)
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+        example.txt
+nothing added to commit but untracked files present (use "git add" to track)
+```
+
+There are three main modes for the `reset` command:
```

Comment:
Everything from this line downward, I think, should. be incorporated above in the "## Syntax" section.

Reply:
> Everything from this line downward, I think, should. be incorporated above in the "## Syntax" section.

will update that , thank you 👍